src/libDirectoryService/ViewChangePostProcessing.cpp

```diff
@@ -287,6 +287,12 @@ void DirectoryService::ProcessViewChangeConsensusWhenDone() {
     return;
   }
 
+  if (!BlockStorage::GetBlockStorage().PutDSCommittee(m_mediator.m_DSCommittee,
```

Comment:
I think also needs to update the same on Lookup's after VC?

Reply:
Lookup and shard node will call function Node::ProcessVCDSBlocksMessage to process VCDS block, and the PutDSCommittee function is called inside it already.

Reply:
Fixed, please review again. @sandipbhoir